google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/apis/GoogleCloudApiSelectorPanelTest.java

```diff
@@ -235,6 +244,37 @@ public void getPanel_withMultipleLibraries_multipleSelected_hasCheckboxesAndPopu
     assertDetailsShownForLibrary(panel.getDetailsPanel(), LIBRARY_1, JAVA_CLIENT_1);
   }
 
+  @Test
+  public void getPanel_withAvailableBomVersions_populatesBomVersions() {
+    when(mavenService.getBomVersions(anyInt()))
+        .thenReturn(
+            ImmutableList.of(newTestVersion("v0"), newTestVersion("v1"), newTestVersion("v2")));
+
+    GoogleCloudApiSelectorPanel panel =
+        new GoogleCloudApiSelectorPanel(ImmutableList.of(), testFixture.getProject());
+
+    JComboBox<Version> bomComboBox = panel.getBomComboBox();
+
+    assertThat(panel.getBomSelectorLabel().isVisible()).isTrue();
+    assertThat(panel.getBomComboBox().isVisible()).isTrue();
+    assertThat(bomComboBox.getItemCount()).isEqualTo(3);
+
+    for (int i = 0; i < bomComboBox.getItemCount(); i++) {
```

Comment:
Nit: remembering Java readability from Nate :) it could be cleaner to move this to a separate test case and test against explicit list of values rather than a loop.

Reply:
sounds good

Reply:
the ghost of Nate haunts us :)

Reply:
I removed the loop. but didn't see much value in moving to a separate test case..lmk if you disagree

Reply:
LGTM